front/src/utils/strings.ts

```diff
@@ -55,3 +55,13 @@ export function onlyDigit(str: string): string {
 export function convertInputStringToNumber(str: string) {
   return str !== '' && !Number.isNaN(str) ? +str : undefined;
 }
+
+/**
+ * Given an UIC number, check if it begins with 87,
+ * if true return the UIC without the 87,
+ * if false return the full UIC
+ * @param uic full UIC and CI code (8 digits)
+ */
+export function formatUicToCi(uic: number) {
+  return uic.toString().startsWith('87') ? uic.toString().slice(2) : uic;
```

Comment:
I don't know if it's better, but I believe you can make a shorter version.

```suggestion
  return uic.replace(/^87/, '');
```

Reply:
yes it's better

Reply:
done

Reply:
Nice, I was not sure it would work 😛 maybe the documentation above the function needs to be updated accordingly 😉